cloud/pkg/cloudhub/servers/httpserver/server.go

```diff
@@ -68,19 +69,18 @@ func StartHTTPServer() {
 
 // getCA returns the caCertDER
 func getCA(w http.ResponseWriter, r *http.Request) {
-	caCertDER := hubconfig.Config.Ca
-	w.Write(caCertDER)
+	w.Write(hubconfig.Config.Ca)
```

Comment:
Please make all changes are related to PR title, recommend to remove this and file in other PR. Same below.